flink-ml-benchmark/src/main/resources/benchmark-conf-kmeans.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Can you add the Apache license header to this file and similar files in the repo? Feel free to see [1] for example.

[1] https://github.com/apache/kafka/blob/trunk/metadata/src/main/resources/common/metadata/TopicRecord.json